perf/istio-install/setup_istio.sh

```diff
@@ -50,7 +50,7 @@ function install_istio() {
         if [[ ! -z "${HELMREPO_URL}" ]];then
           helmrepo="${HELMREPO_URL}"
         fi
-        helm repo add istio.io "${helmrepo}"
```

Comment:
we can just remove this whole helm block, we do all this stuff and then never use it

Reply:
ok good point.

Reply:
Remove section.

Reply:
Does not work without this change, fails with this:
kubectl apply -f /home/prow/go/src/istio.io/tools/perf/istio-install/tmp/master-20190424-09-16/istio-init/files/
error: the path "/home/prow/go/src/istio.io/tools/perf/istio-install/tmp/master-20190424-09-16/istio-init/files/" does not exist

Reply:
Fixed in https://github.com/istio/tools/pull/111 @utka

Reply:
Thank you!